components/advocates/TheAdvocatesHeader.vue

```diff
@@ -22,3 +22,11 @@ import TypewriterEffect from '~/components/ui/TypewriterEffect.vue'
 @Component({ components: { ThePageHeader, TypewriterEffect } })
 export default class extends Vue { }
 </script>
+
+<style lang="scss">
+.advocates-header {
+  .page-header__title {
+    max-width: initial;
+  }
+}
+</style>
```

Comment:
This is not a join section change, so I think it is better to keep these changes in a separate PR.